app/javascript/react/components/ThresholdConfigurator/ThresholdConfigurator.style.tsx

```diff
@@ -0,0 +1,112 @@
+import styled from "styled-components";
+
+import * as colors from "../../assets/styles/colors";
+
+const Container = styled.div`
+  margin: 50px;
+  background-color: ${colors.white};
+  position: relative;
+  width: 90%;
+  height: 70px;
+`;
+
+const RangeInput = styled.input<{
+  $min: number;
+  $max: number;
+  $firstThumbPos: number;
+  $secondThumbPos: number;
+  $thirdThumbPos: number;
+  $sliderWidth: number;
+}>`
+  width: 100%;
+  position: absolute;
+  top: 0;
+  height: 8px;
+  background: transparent;
+  -webkit-appearance: none;
+  appearance: none;
+  background: transparent;
```

Comment:
repetition here, it is presented above already

Reply:
Removed!